### PR TITLE
fix(curriculum): Added an example for Math.abs()

### LINKED ISSUE #53487: Add code example for `Math.abs` introduction in step 81 calorie counter
### Describe the Issue

Whenever we introduce a new concept, we should include a code example.
Sometimes we do this and sometimes we don't which creates inconsistency in the curriculum. 
To fix this let's update the description to include a code example.

## updated description for step 81 
````md
# --description--

When the user has a calorie deficit, the `remainingCalories` value will be negative. You don't want to display a negative number in the result string.

`Math.abs()` is a built-in JavaScript method that will return the absolute value of a number.

```js
const num = -5;
Math.abs(num); // 5
```

In your `span` text, wrap your `remainingCalories` reference in `Math.abs()` to ensure that the value is positive.
````

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-form-validation-by-building-a-calorie-counter/step-81

### Your code

see explanation above

### Expected behavior

see explanation above

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by SaifullahBijapur**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

Closes #53487

### CODE REVIEW

**Review by gikf — COMMENTED** [review]
Note for future - it is recommended to make changes on a separate branch, than the `main` branch. See more about it in point 3 at https://contribute.freecodecamp.org/#/how-to-contribute-to-the-codebase?id=contributing-to-the-codebase.

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e769df38c92635c158ba.md [hunk 1]
```diff
@@ -9,7 +9,12 @@ dashedName: step-81
 
 When the user has a calorie deficit, the `remainingCalories` value will be negative. You don't want to display a negative number in the result string.
 
-`Math.abs()` is a built-in JavaScript method that will return the absolute value of a number. In your `span` text, wrap your `remainingCalories` reference in `Math.abs()` to ensure that the value is positive.
```
Comment (gikf) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e769df38c92635c158ba.md:12]: Could you re-add the second sentence below added code example? Without it description doesn't say what needs to be done in this step.

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e769df38c92635c158ba.md [hunk 2]
```diff
@@ -9,7 +9,12 @@ dashedName: step-81
...
+`Math.abs()` is a built-in JavaScript method that will return the absolute value of a number.
```
Comment (jdwilkin4) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e769df38c92635c158ba.md:12]: It looks like there was a misunderstanding on how the code example should be added.
As mentioned earlier, you shouldn't delete the instructions for the challenge.
The goal here is to just add a code example for `Math.abs`.

Here should be the correct complete description.
Please copy this and make those changes 👍 

````md
# --description--

When the user has a calorie deficit, the `remainingCalories` value will be negative. You don't want to display a negative number in the result string.

`Math.abs()` is a built-in JavaScript method that will return the absolute value of a number.

```js
const num = -5;
Math.abs(num); // 5
```

In your `span` text, wrap your `remainingCalories` reference in `Math.abs()` to ensure that the value is positive.
````

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

**Review by jdwilkin4 — APPROVED** [review]
much better 👍

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e769df38c92635c158ba.md [hunk 3]
```diff
@@ -4,13 +4,18 @@ title: Step 81
 challengeType: 0
 dashedName: step-81
 ---
```
Comment (gikf) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e769df38c92635c158ba.md:6]:
```suggestion
---

```

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e769df38c92635c158ba.md [hunk 4]
```diff
@@ -4,13 +4,18 @@ title: Step 81
...
 
-`Math.abs()` is a built-in JavaScript method that will return the absolute value of a number. In your `span` text, wrap your `remainingCalories` reference in `Math.abs()` to ensure that the value is positive.
+`Math.abs()` is a built-in JavaScript method that will return the absolute value of a number.
+
+```js
+const num = -5;
+Math.abs(num); // 5
+```
 
+In your `span` text, wrap your `remainingCalories` reference in `Math.abs()` to ensure that the value is positive.
```
Comment (gikf) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e769df38c92635c158ba.md:18]:
```suggestion
In your `span` text, wrap your `remainingCalories` reference in `Math.abs()` to ensure that the value is positive.

```

**Review by gikf — COMMENTED** [review]
Just slight nitpicking, please keep empty lines between sections for consistency.

**Comment by jdwilkin4** [comment]
Hi @SaifullahBijapur ! 

It looks like some of the automated checks are failing.
I would suggest updating your branch with the latest changes from main to see if that resolves it

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e769df38c92635c158ba.md (hunk 3 above)
Comment (SaifullahBijapur) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e769df38c92635c158ba.md:6]: ok, I will take care of that

**Review by gikf — APPROVED** [review]

**Comment by gikf** [comment]
Hi @SaifullahBijapur. Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**